docs/src/_guide/advanced-topics.md

```diff
@@ -14,6 +14,39 @@ Most of Meltano's features are available without installing any additional packa
 * `gcs` - Support for using Google Cloud Storage as a [state backend](/concepts/state_backends)
 * `azure` - Support for using Azure Blob Storage as a [state backend](/concepts/state_backends)
 
+## Extension Developer Kit (EDK)
+
+Meltano extensions are thin wrappers around applications that allow you to integrate existing data tools with Meltano.
```

Comment:
```suggestion
Meltano extensions are lightweight shims that allow you to integrate existing data tools with Meltano.
```

Thoughts on this language? I don't want to give the impression that we fully wrap the tool and that our Airflow is somehow different from a "pure" Airflow